### ISSUE TITLE
localhost

### ISSUE THREAD

**Issue by rootseire** [okfn/timemapper#176]
Cloned a local version of this and tested it, getting the following error:

C\wampserver\...\file.js:116
if(!path.existsSync(self.file))

TypeError: undefined is not a function at File.loadSync ...lib\nconf\stores\fle.js:116:13

Any reason why this shows up?

**Comment by rgrp** [comment]
@rootseire sadly no idea as we have not installed on windows. It looks like any issue in nconf library.

**Comment by rootseire** [comment]
Fair enough, thanks, ive got through some hoops while at it yesterday (modules was also throwing up errors) but hopefully this will be the last one!

**Comment by rootseire** [comment]
I got this working on Windows 7 finally, by tweaking file.js

change if (!path.existsSync(self.file)) {s to if (!fs.existsSync(self.file)) { in file.js line 116

Should fire up then.

I'd like to implement the language option - similar to that on TimelineJS when you create a timeline that you can choose the language aswell and it comes up in the url - would it be possible to add this code? I've noticed that this doesn't work in the urls for Timemapper. So in my own version I have added a dropdown for languages and was thinking that it would be a matter of using the js files like on TimelineJS -  but think it would be better to have it loading dynamically?
